frontend/app/(defaultLayout)/helpers.tsx

```diff
@@ -0,0 +1,24 @@
+import { ifixitOriginFromHost } from '@helpers/path-helpers';
+import { GetServerSidePropsContext } from 'next';
+import { headers } from 'next/headers';
+
+export function iterableToObj(
+   iterator: IterableIterator<[string, string]>
+): Record<string, string> {
+   const result: Record<string, string> = {};
+   let item = iterator.next();
+   while (!item.done) {
+      const [key, value] = item.value;
+      result[key] = value;
+      item = iterator.next();
+   }
+   return result;
+}
+
+export function ifixitOriginFromHostWrapper(): string {
```

Comment:
I'm also a little confused why we have this. The iFixit origin that should be used is in an ENV variable I thought.